build/push-ci-build.sh

```diff
@@ -32,5 +32,11 @@ KUBE_GCS_LATEST_CONTENTS=${LATEST}
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "$KUBE_ROOT/build/common.sh"
 
-kube::release::gcs::release
-kube::release::gcs::publish_latest
+MAX_ATTEMPTS=3
+attempt=0
+while [[ ${attempt} -lt ${MAX_ATTEMPTS} ]]; do
+  kube::release::gcs::release && kube::release::gcs::publish_latest && break || true
```

Comment:
OTOH, this is doing exactly the same thing as pressing the "retry build" button on Jenkins, which usually seems to work.